scripting/tf2utils.sp

```diff
@@ -491,6 +521,12 @@ void OnPreThinkPost(int client) {
 	}
 }
 
+static MRESReturn DHook_StartLagCompensation(Address address)
+{
+	addr_CLagCompensationManager = address;
+	return MRES_Ignored;
+}
+
```

Comment:
Pretty sure you can get `lagcompensation` via symbol on Linux and `CTFPistol_ScoutPrimary::Push + 46h` on Windows (string xref "Weapon_Hands.PushImpact").  DHooks is largely unnecessary otherwise.

Reply:
Hopefully the latest changes are good enough